doc/dev/dev_setup.md

```diff
@@ -20,7 +20,7 @@ or execute the various commands available in the toolbox.
     cd azure-sdk-for-python
     ```
 
-2.  Create a [virtual environment](https://docs.python.org/3/tutorial/venv.html)
+2.  Create a [virtual environment][virtual_environment]
```

Comment:
You may need to add a newline or period at the end of the sentence to separate this and the next line.